src/CreditsSlider.js

```diff
@@ -1,25 +1,42 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {makeStyles} from "@material-ui/core/styles";
+import Slider from '@material-ui/core/Slider';
+import Typography from '@material-ui/core/Typography';
 
-const useStyles = makeStyles((theme) => ({
-    root: {
-        backgroundColor: "green",
-    }
-}));
+const useStyles = makeStyles((theme) => ({}));
 
 /** Assigned to Dominik */
-const CreditsSlider = () => {
+const CreditsSlider = ({setValue, value, maxvalue}) => {
```

Comment:
byl bych pro držet se konvence `onChange` místo `setValue`. Takto to mají všechny inputy v material ui.